src/cmd/build.js

```diff
@@ -104,6 +105,46 @@ export async function getDefaultLocalizedName(
   return Promise.resolve(extensionName);
 }
 
+// https://stackoverflow.com/a/22129960
+function resolve(prop, obj, separator = '.'): string {
+  var properties = Array.isArray(prop) ? prop : prop.split(separator);
+  var value = properties.reduce((prev, curr) => prev && prev[curr], obj);
+  if (typeof value !== 'string') {
```

Comment:
Nit, `manifest_version` is a number, maybe we can be a bit more permissive and allow "number" as well.